x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/task_runner.ts

```diff
@@ -309,9 +309,9 @@ export default function createBackfillTaskRunnerTests({ getService }: FtrProvide
       // check timestamps in alert docs
       for (const alert of alertDocsBackfill1) {
         const source = alert._source!;
-        expect(source[ALERT_START]).to.eql(scheduleResult[0].schedule[0].run_at);
-        expect(source[ALERT_LAST_DETECTED]).to.eql(scheduleResult[0].schedule[0].run_at);
-        expect(source[TIMESTAMP]).to.eql(scheduleResult[0].schedule[0].run_at);
+        expect(source[ALERT_START]).not.to.eql(scheduleResult[0].schedule[0].run_at);
+        expect(source[ALERT_LAST_DETECTED]).not.to.eql(scheduleResult[0].schedule[0].run_at);
+        expect(source[TIMESTAMP]).not.to.eql(scheduleResult[0].schedule[0].run_at);
```

Comment:
I think we discussed testing that `TIMESTAMP` matched `ALERT_START` but then discussed how they would be different because they both are set using `new Date()` instead of a fixed timestamp :). So I think we can just add an assertion that they match the timestamp pattern

```
expect(source[ALERT_START]).to.match(timestampPattern);
expect(source[ALERT_LAST_DETECTED]).to.match(timestampPattern);
```